# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/wrench/reftests/filters/svgfe-blendnormal-linear.yaml
# Commit: c0f58fd74e56
# Full Hash: c0f58fd74e56b24f67fccae08097d151e21a0f80
# Author: Ashley Hale <ahale@mozilla.com>
# Date: 2024-05-09 16:15:25
# Regressor Bug: 1824502
# File Overlap Count: 1
# Description:
#   Bug 1824502 - Implement native SVG filter rendering in WebRender r=gw
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D209646
# ==============================================================================

diff -r 3e2ec3a3b738 -r c0f58fd74e56 gfx/wr/wrench/reftests/filters/svgfe-blendnormal-linear.yaml
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/gfx/wr/wrench/reftests/filters/svgfe-blendnormal-linear.yaml	Thu May 09 04:09:00 2024 +0000
@@ -0,0 +1,21 @@
+---
+root:
+  items:
+    - type: stacking-context
+      bounds: [0, 0, 120, 120]
+      filters:
+        - svgfe: SourceGraphic
+        - svgfe: SourceAlpha
+        - svgfe: flood
+          linear: true
+          subregion: [10, 10, 100, 100]
+          color: [192, 96, 128, 1]
+        - svgfe: blendnormal
+          linear: true
+          subregion: [10, 10, 100, 100]
+          in: 0
+          in2: 2
+      items:
+        - type: rect
+          bounds: [10, 10, 100, 100]
+          color: [128, 64, 32, 0.5]